model/qti/feedback/Feedback.php

```diff
@@ -90,7 +95,7 @@ public function isIdentifierAvailable($newIdentifier)
                 $collection = $relatedItem->getIdentifiedElements();
 
                 try {
-                    $feedback = $collection->getUnique($newIdentifier, 'oat\\taoQtiItem\\model\\qti\\feedback\\Feedback');
+                    $feedback = $collection->getUnique($newIdentifier, static::class);
```

Comment:
I don't really know if it makes a difference, but just to be safe, kindly consider replacing the `static` reference with `self`, in order to keep the behavior as it was.
```suggestion
                    $feedback = $collection->getUnique($newIdentifier, self::class);
```

Reply:
Actually it does not matter. But I don't mind.